src/Microsoft.AspNetCore.Server.Kestrel.Core/KestrelServerLimits.cs

```diff
@@ -134,6 +137,28 @@ public int MaxRequestHeaderCount
         }
 
         /// <summary>
+        /// Gets or sets the maximum allowed size of any request body in bytes.
+        /// When set to null, the maximunm request body size is unlimited.
+        /// This limit has no affect on upgraded connections which are always unlmited.
+        /// This can be overridden by <see cref="Internal.Http.IHttpMaxRequestBodySizeFeature"/>
+        /// </summary>
+        /// <remarks>
+        /// Defaults to null (unlimited).
+        /// </remarks>
+        public long? MaxRequestBodySize
```

Comment:
Add tests for default value, valid values and invalid values in `KestrelServerLimitsTests`.